docs/mathematics/lucky-numbers.mdx

```diff
@@ -0,0 +1,336 @@
+---
+title: Lucky Number
+slug: /mathematics/lucky-numbers
+---
+
+## Introduction
+
+Lucky numbers are a subset of integers that are generated by the result of a sieving process. The sieving process eliminates numbers based on their position in the remaining set.
+
+## Explanation
+
+Let us have one example: 
+
+```
+- Start with a list of integers beginning with 1.
+  1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, . . . 
+
+- Now, delete every second number from the above set.
+  1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21, 23, 25, 27, 29, 31, . . .
+
+- The first surviving number after 1 is 3. Delete every third number from the above set.
+  1, 3, 7, 9, 13, 15, 19, 21, 25, 27, 31, . . .
+
+- The next surviving number is seven. Delete every seventh number from the above set.
+  1, 3, 7, 9, 13, 15, 21, 25, 27, 31, . . .
+
+- The next surviving number is nine. Delete every ninth number from the above set.
+  1, 3, 7, 9, 13, 15, 21, 25, 31, . . .
+
+- Similarly, the next surviving number is thirteen. The above process is continued.
+
+- The number which doesn't get deleted due to the above process is a LUCKY NUMBER.
+
+- First few lucky numbers are: 1, 3, 7, 9, 13, 15, 21, 25, . . .  
+
```

Comment:
A better and more consumable example would be appreciated. Start with example first and then proceed with explanation